.github/workflows/publish.yaml

```diff
@@ -0,0 +1,88 @@
+name: Publish
+on:
+  push:
+    tags: '*'
+
+permissions:
+  contents: write
+  packages: write
+
+jobs:
+  publish-docker-image:
+    name: Publish Docker image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.17.0
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v2.2.2
+        id: pnpm-install
+        with:
+          version: 7
+          run_install: false
+
+      - name: Get pnpm store directory
+        id: pnpm-cache
+        run: |
+          echo "::set-output name=pnpm_cache_dir::$(pnpm store path)"
+      - name: Setup pnpm cache
+        uses: actions/cache@v3
+        with:
+          path: ${{ steps.pnpm-cache.outputs.pnpm_cache_dir }}
+          key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-pnpm-store-
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile --prefer-offline
+
+      - name: Set prod env
+        run: mv apps/cugetreg-web/.env.prod apps/cugetreg-web/.env
+
+      - name: Extract app and version
+        id: meta
+        env:
+          TAG: ${{ github.ref_name }}
+        run: |
+          echo "::set-output name=APP::${TAG%@*}"
+          echo "::set-output name=VERSION::${TAG##*@}"
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Docker login
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ github.token }}
+
+      - name: Build Docker images
+        run: npx nx run ${{ steps.meta.outputs.APP }}:docker-build
```

Comment:
use [run-many](https://nx.dev/packages/nx/documents/run-many) command instead

```
nx run-many --parallel --target=serve --projects=${{ steps.meta.outputs.APP }}
```

Note: `APP` should follow by this format, `app1,app2,app3`

Reply:
This workflow will only run for one app at a time.